src/main/resources/chess_board.fxml

```diff
@@ -0,0 +1,158 @@
+    <Label fx:id="square28" GridPane.columnIndex="5" GridPane.rowIndex="4"/>
+    <Label fx:id="square29" GridPane.columnIndex="6" GridPane.rowIndex="4"/>
+    <Label fx:id="square30" GridPane.columnIndex="7" GridPane.rowIndex="4"/>
+    <Label fx:id="square31" GridPane.columnIndex="8" GridPane.rowIndex="4"/>
+
+    <Label fx:id="square32" GridPane.columnIndex="1" GridPane.rowIndex="5"/>
+    <Label fx:id="square33" GridPane.columnIndex="2" GridPane.rowIndex="5"/>
+    <Label fx:id="square34" GridPane.columnIndex="3" GridPane.rowIndex="5"/>
+    <Label fx:id="square35" GridPane.columnIndex="4" GridPane.rowIndex="5"/>
+    <Label fx:id="square36" GridPane.columnIndex="5" GridPane.rowIndex="5"/>
+    <Label fx:id="square37" GridPane.columnIndex="6" GridPane.rowIndex="5"/>
+    <Label fx:id="square38" GridPane.columnIndex="7" GridPane.rowIndex="5"/>
+    <Label fx:id="square39" GridPane.columnIndex="8" GridPane.rowIndex="5"/>
+
+    <Label fx:id="square40" GridPane.columnIndex="1" GridPane.rowIndex="6"/>
+    <Label fx:id="square41" GridPane.columnIndex="2" GridPane.rowIndex="6"/>
+    <Label fx:id="square42" GridPane.columnIndex="3" GridPane.rowIndex="6"/>
+    <Label fx:id="square43" GridPane.columnIndex="4" GridPane.rowIndex="6"/>
+    <Label fx:id="square44" GridPane.columnIndex="5" GridPane.rowIndex="6"/>
+    <Label fx:id="square45" GridPane.columnIndex="6" GridPane.rowIndex="6"/>
+    <Label fx:id="square46" GridPane.columnIndex="7" GridPane.rowIndex="6"/>
+    <Label fx:id="square47" GridPane.columnIndex="8" GridPane.rowIndex="6"/>
+
+    <Label fx:id="square48" GridPane.columnIndex="1" GridPane.rowIndex="7"/>
+    <Label fx:id="square49" GridPane.columnIndex="2" GridPane.rowIndex="7"/>
+    <Label fx:id="square50" GridPane.columnIndex="3" GridPane.rowIndex="7"/>
+    <Label fx:id="square51" GridPane.columnIndex="4" GridPane.rowIndex="7"/>
+    <Label fx:id="square52" GridPane.columnIndex="5" GridPane.rowIndex="7"/>
+    <Label fx:id="square53" GridPane.columnIndex="6" GridPane.rowIndex="7"/>
+    <Label fx:id="square54" GridPane.columnIndex="7" GridPane.rowIndex="7"/>
+    <Label fx:id="square55" GridPane.columnIndex="8" GridPane.rowIndex="7"/>
+
+    <Label fx:id="square56" GridPane.columnIndex="1" GridPane.rowIndex="8"/>
+    <Label fx:id="square57" GridPane.columnIndex="2" GridPane.rowIndex="8"/>
+    <Label fx:id="square58" GridPane.columnIndex="3" GridPane.rowIndex="8"/>
+    <Label fx:id="square59" GridPane.columnIndex="4" GridPane.rowIndex="8"/>
+    <Label fx:id="square60" GridPane.columnIndex="5" GridPane.rowIndex="8"/>
+    <Label fx:id="square61" GridPane.columnIndex="6" GridPane.rowIndex="8"/>
+    <Label fx:id="square62" GridPane.columnIndex="7" GridPane.rowIndex="8"/>
+    <Label fx:id="square63" GridPane.columnIndex="8" GridPane.rowIndex="8"/>
+
+    <fx:define>
+
+        <ArrayList fx:id="squares">
+            <fx:reference source="square0" styleClass="light"/>
```

Comment:
You should be able to set "styleClass" in the xml for the Label, not in the reference.